kafka-rest/src/main/java/io/confluent/kafkarest/controllers/SchemaManagerImpl.java

```diff
@@ -128,20 +150,35 @@ private RegisteredSchema getSchemaFromSchemaVersion(
     String actualSubject =
         subject.orElse(getSchemaSubjectUnsafe(topicName, isKey, subjectNameStrategy));
 
-    Schema schema =
-        schemaRegistryClient.getByVersion(
-            actualSubject, schemaVersion, /* lookupDeletedSchema= */ false);
-
-    ParsedSchema parsedSchema =
-        EmbeddedFormat.forSchemaType(schema.getSchemaType())
-            .getSchemaProvider()
-            .parseSchema(schema.getSchema(), schema.getReferences(), /* isNew= */ false)
-            .orElseThrow(
-                () ->
-                    Errors.invalidSchemaException(
-                        String.format(
-                            "Error when fetching schema by version. subject = %s, version = %d",
-                            actualSubject, schemaVersion)));
+    Schema schema;
+    try {
+      schema =
+          schemaRegistryClient
+              .get()
+              .getByVersion(actualSubject, schemaVersion, /* lookupDeletedSchema= */ false);
+    } catch (RuntimeException restClientException) {
```

Comment:
nit: `e`

We usually only name exceptions when we have multiple in the same context. Otherwise, the idiomatic is just to use `e`.